packages/@o3r/rules-engine/src/engine/operator/operators/basic.operators.ts

```diff
@@ -91,7 +91,22 @@ export const isUndefined: UnaryOperator<any> = {
   evaluator: (input) => input === undefined || input === null
 };
 
+/**
+ * Check if the text variable matches the specified RegExp pattern
+ *
+ * @title matches the pattern
+ */
+export const matchesPattern: Operator<string, string> = {
+  name: 'matchesPattern',
+  evaluator: (value, pattern) => {
+    const regExp = new RegExp(pattern);
```

Comment:
Not as simple: `new RegExp('/a/i')` is not equivalent to `/a/i` but `/\/a\/i/`.
You will need to handle the 2 `/` of the regexp and the options given in the end (that are the second parameter of the `new RegExpp()`)